WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -26,6 +26,7 @@ public async Task<InputRegistrationResponse> RegisterInputAsync(InputRegistratio
 		}
 		catch (Exception ex) when (IsUserCheating(ex))
 		{
+			Logger.LogInfo($"{request.Input} is cheating: {ex.Message}");
```

Comment:
Maybe add the `roundId` as well to the beggining?
```suggestion
			Logger.LogInfo($"{request.RoundId}: {request.Input} is cheating: {ex.Message}");
```

Reply:
No. RoundId is useless.